src/routes/balances.rs

```diff
@@ -5,15 +5,25 @@ use crate::utils::context::Context;
 use crate::utils::errors::ApiResult;
 use rocket::response::content;
 
-#[get("/v1/safes/<safe_address>/balances/<fiat>")]
+#[get("/v1/safes/<safe_address>/balances/<fiat>?<trusted>&<exclude_spam>")]
 pub fn get_balances(
     context: Context,
     safe_address: String,
     fiat: String,
+    trusted: Option<bool>,
+    exclude_spam: Option<bool>,
 ) -> ApiResult<content::Json<String>> {
+    let trusted = trusted.unwrap_or(false);
+    let exclude_spam = exclude_spam.unwrap_or(false);
```

Comment:
Not sure if we should default to `true` here

Reply:
I matched the values in the tx service for the `/safes/{address}/balances/usd/` endpoint

Reply:
Yes, but the clients should not show spam tokens by default, therefore it would make more sense to have `true` as a default for the client gateway.

Reply:
Np, I can change it to `true`, I will create an issue, just for documentation's sake.